era_5g_interface/h264_decoder.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from pathlib import Path
+
+from av import Packet
+from av.codec import CodecContext
+from av.video.codeccontext import VideoCodecContext
+
+Path("output").mkdir(parents=True, exist_ok=True)
+
+logger = logging.getLogger("H264 decoder")
+
+
+class H264Decoder:
+    def __init__(self, fps: float, width: int, height: int) -> None:
+        self.decoder: VideoCodecContext = CodecContext.create("h264", "r")
+        self.decoder.width = width
+        self.decoder.height = height
+        self.decoder.framerate = fps
+        self.decoder.pix_fmt = "yuv420p"
+
+    def decode_packet_data(self, packet_data, format: str = "bgr24"):
+        packet = Packet(packet_data)
+        # try:
```

Comment:
Remove commented code or add an explanation of why it is commented (if it is important stuff).